conformance/metadata-fhirr4-service/README.md

```diff
@@ -0,0 +1 @@
+# FHIR R4 Metadata Service
```

Comment:
Shall we keep a task to have descriptive readme files for all pre-built services on how to setup etc?